modules/virt-installing-virtctl-client-web.adoc

```diff
@@ -3,22 +3,16 @@
 // virt/install/virt-installing-virtctl.adoc
 
 [id="virt-installing-virtctl-client-web_{context}"]
-= Installing the virtctl client from the web console
+= Installing the virtctl client using the web console
 
-You can download the `virtctl` client from the Red Hat Customer Portal, which is linked to in your {VirtProductName} web console in the *Command Line Tools* page.
+Install the `virtctl` client from the {VirtProductName} web console.
 
 .Prerequisites
 
-* You must have an activated {product-title} subscription to access the download page on the Customer Portal.
+* You must have downloaded the `virtctl` client using the web console or CLI.
 
 .Procedure
```

Comment:
This section must be updated as well. For linux, "tar.gz" must be extracted as explained here. For Mac+Linux, zip file must be extracted.  Will be similar to https://docs.openshift.com/container-platform/4.8/serverless/cli_tools/installing-kn.html#installing-cli-macosx_installing-kn